packages/next/server/config.ts

```diff
@@ -74,7 +83,7 @@ function assignDefaults(userConfig: { [key: string]: any }) {
         typeof value === 'object' &&
         Object.keys(value).length > 0
       ) {
-        experimentalWarning()
+        experimentalWarning(configFileName, Object.keys(value))
```

Comment:
Does this mean that all values are read instead of the supported experimental features? It seems this would cause the following code to display the keys:

```
module.exports = {
  experimental: {
    randomFeatureThatDoesNotExist: true
  }
}
```

Reply:
Nice catch! This was fixed in #37755